index.bs

```diff
@@ -475,7 +475,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     1. If |extension| is not supported by this client platform, then [=continue=].
 
     1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, [=continue=].
+        algorithm returned an error, return [=a promise rejected with=] a {{DOMException}} whose name is
+        "{{TypeError}}", and terminate this algorithm.
```

Comment:
This makes sense to me, although I don't have a strong opinion.

If an extension is critical for security, the RP needs to check for it in the response regardless, because of the previous "if not supported then continue" step. If the extension is optional, you still want some way for developers to notice that their argument is malformed and so won't work on any browser. Throwing a TypeError here is plausible. Checking the "exts" extension and then for a missing response for supported extensions would also work, but takes a lot more developer effort.